fixcoveragefilepaths.py

```diff
@@ -0,0 +1,67 @@
+import xml.etree.ElementTree as xml
```

Comment:
Could you add a brief high-level description why this file is necessary and who it solves / works around some problem? Is there a known bug report or similar which we can link to?